.azp/validate.yml

```diff
@@ -18,228 +18,244 @@ variables:
   ctest_stride: 4
   mpiexec_timeout: 21600
 
+# a temporary empty job so that CI will pass while the actual tests are disabled
```

Comment:
I assume we plan on adding back validation tests later? Similar to the docs?

Reply:
Yes, we will convert existing validation tests over to pytest in another PR.